src/vm/i386/excepx86.cpp

```diff
@@ -3769,10 +3783,27 @@ AdjustContextForVirtualStub(
     return TRUE;
 }
 
+#ifndef DACCESS_COMPILE
+#ifndef WIN64EXCEPTIONS
 #ifdef FEATURE_PAL
+EXTERN_C
+_Unwind_Reason_Code
+UnhandledExceptionHandlerUnix(
```

Comment:
Yes, I thought that here is better place if we will remove all the WIN32EXCEPTIONS-related features from x86/Linux port.